Azure/communication-ui-library-android
# [Feature] Let Talkback read confirm leave overlay properly

## Purpose
When confirm leave overlay is up, Talkback should focus on the buttons on the overlay. 
When confirm leave overlay is up and swiping left or right to navigate the UI elements, Talkback should not focus on the UI elements underneath.

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[X] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

Turn on screen reader(eg. talkback on pixel)
run the app to join group call with remote participants
using swipe to focus on leave call button on the bottom bar
double click to open confirm leave overlay
verify: 1. the first element being focused is blue button on the overlay.
            2. swiping left or right, screen reader should not focus on any UI elements under confirm leave overlay
